tests/network-ovn

```diff
@@ -216,11 +216,13 @@ ovn_basic_tests() {
     lxc network unset dummy ipv6.routes.anycast --project default
 
     lxc init "${IMAGE}" u1 --project testovn -s default
-    lxc config device add u1 eth0 nic network=ovn-virtual-network name=eth0 --project testovn
 
-    echo "==> Record NAT rules count before u1 started"
-    natRulesBefore=$(ovn-nbctl --bare --format=csv --column=external_ip,logical_ip,type find nat | wc -l)
+    echo "==> Record NAT rules count before u1 eth0 NIC added"
+    out="$(ovn-nbctl --bare --format=csv --column=external_ip,logical_ip,type find nat)"
+    natRulesBefore="$(wc -l <<< "${out}")"
+    ovn-nbctl --bare --format=csv --column=external_ip,logical_ip,type find nat
```

Comment:
That line is extraneous because the one producing the `out` value would have been shown already thanks to `set -x`.